main/src/ca/uwaterloo/flix/language/ast/Type.scala

```diff
@@ -368,12 +377,24 @@ object Type {
     * Constructs the tuple type (A, B, ...) where the types are drawn from the list `ts`.
     */
   def mkTuple(ts: List[Type]): Type = {
+    if (ts.lengthIs <= 1) {
```

Comment:
What is `lengthIs`??

Reply:
`lenthIs` returns a `SizeCompareOps` object. It allows for a `O(min(length, val))` length comparison rather than a `O(length)` comparison which would be the case for `ts.length <= 1`. Probably not making a big difference here, but it's a "best practice" for Scala as I understand.

Reply:
Thanks! You learn something new every day!